wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -500,15 +500,20 @@ extension ConversationCreationController {
     }
 }
 
-extension Feature.MLS.Config.MessageProtocol {
-    var toMessageProtocol: MessageProtocol {
```

Comment:
@jullianm I removed `toMessageProtocol`, and have only one `defaultProtocol` helper property

Reply:
Ok, sorry I thought this was legacy code